lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -183,34 +183,53 @@ const Reportback = {
    * @param  {array} data  Array of Kudos objects from API.
    * @return {array}
    */
-  getReactions: function(data) {
-    let reactions = [];
-
-    forEach(this.reactions.approved, (term) => {
-      const termId = this.reactions.terms[term];
+  getReactions: function(data, reactionFromRogue) {
+    if (reactionFromRogue) {
```

Comment:
@DFurnes SECOND COMMENT! Since we're only passing one reaction here, I wrote some code that will give the same response but with Rogue data.